README.md

```diff
@@ -249,7 +257,200 @@ AdyenCardComponent(
+final sessionCheckout = await AdyenCheckout.session.create(
+  sessionId: sessionResponse.id,
+  sessionData: sessionResponse.sessionData,
+  configuration: applePayComponentConfiguration,
+);
+```
+4. Get the Apple Pay payment method from the `sessionCheckout` object.
+5. Use the Apple Pay component widget:
+```dart
+AdyenApplePayComponent(
+  configuration: applePayComponentConfiguration,
+  paymentMethod: applePayPaymentMethod,
+  checkout: sessionCheckout,
+  style: ApplePayButtonStyle(width: 200, height: 48),
+  onPaymentResult: (paymentResult) {
+    //Handle the payment result
+  },
+),
+``` 
+<br>
+
+### Apple Pay Component with Advanced flow
+
+1. From your backend, make a [`/paymentMethods`](https://docs.adyen.com/api-explorer/Checkout/71/post/paymentMethods) request.
+2. Get the Apple Pay payment method by filtering the payment methods list.
+3. Create the `ApplePayComponentConfiguration`. It requires the `ApplePayConfiguration` which contains your merchant id and merchant name. You can also provide optional properties for an enhanced Apple Pay use case.
+
+```dart
+final ApplePayConfiguration applePayConfiguration = ApplePayConfiguration(
+  merchantId: MERCHANT_ID,
+  merchantName: MERCHANT_NAME,
+);
+
+final ApplePayComponentConfiguration applePayComponentConfiguration = ApplePayComponentConfiguration(
+  // Change the environment to live when you are ready to accept real payments.
+  environment: Environment.test
+  clientKey: CLIENT_KEY,
+  countryCode: COUNTRY_CODE,
+  amount: AMOUNT,
+  applePayConfiguration: applePayComponentConfiguration,
+);
+```
+4. Create an `AdvancedCheckoutPreview` providing two callbacks:
+- `onSubmit`: from your backend, make a [`/payments`](https://docs.adyen.com/api-explorer/Checkout/latest/post/payments) request. The callback returns two parameters:
+  - `data`: payment data that needs to be forwarded.
+  - `extra`: extra information (e.g. shipping address) in case it is specified in the Apple Pay configuration. Can be null if not needed.
+- `onAdditionalDetails`: from your backend, make a [`/payments/details`](https://docs.adyen.com/api-explorer/Checkout/71/post/payments/details)
+```dart
+final AdvancedCheckoutPreview advancedCheckout = AdvancedCheckoutPreview(
+  onSubmit: YOUR_ON_SUBMIT_CALL,
+  onAdditionalDetails: YOUR_ON_ADDITIONAL_DETAILS_CALL,
+);
+```
+
+5. Create the Apple Pay component widget:
+```dart
+AdyenApplePayComponent(
+  configuration: applePayComponentConfiguration,
+  paymentMethod: paymentMethod,
+  checkout: advancedCheckout,
+  style: ApplePayButtonStyle(width: 200, height: 48),
+  onPaymentResult: (paymentResult) {
+      //Handle the payment result
+  },
+),
+``` 
+<br>
+
+### Google Pay Component with Sessions flow
+1. From your backend, make a [`sessions`](https://docs.adyen.com/api-explorer/Checkout/71/post/sessions) request. The response contains:
+- `sessionData`: the payment session data.
+- `id`: a unique identifier for the session data.
+- The request body.
```

Comment:
Same question about the request body as above, do they have to pass that to the app?